script.js

```diff
@@ -1,38 +1,8 @@
-let awesomeBooks = [];
-let bookIncrement = 0;
-function Book(bookTitle, bookAuthor, bookId) {
-  this.bookTitle = bookTitle;
-  this.bookAuthor = bookAuthor;
-  this.bookId = bookId;
-}
-
-function getBooks() {
-  const tempStorage = JSON.parse(localStorage.getItem('awesomeBooks'));
-  if (tempStorage) {
-    awesomeBooks = tempStorage;
-  }
-}
-
-getBooks();
-
-function getBookId() {
-  let tempBoy = 0;
-  try {
-    tempBoy = awesomeBooks[awesomeBooks.length - 1].bookId + 1;
-  } catch {
-    tempBoy = 0;
-  } finally {
-    bookIncrement = tempBoy;
-  }
-}
-getBookId();
+/* eslint-disable max-classes-per-file, no-use-before-define */
```

Comment:
[Optional]
You can use multiple files and [import your classes](https://www.digitalocean.com/community/tutorials/js-modules-es6) to avoid this error. Suppressing linter errors should be the last option to choose.

Reply:
Hi Sinan,

Are you talking about Webpack? As we didn't learn webpack yet.
Or do you know a way without webpacker? 😄